---
float32[] exposure_bounds
bool supports_manual_exposure
